hack/make-rules/test-cmd.sh

```diff
@@ -127,6 +127,8 @@ if [[ ${WHAT} == "" || ${WHAT} =~ .*kubeadm.* ]] ; then
   make all -C "${KUBE_ROOT}" WHAT=cmd/kubeadm
   # unless the user sets KUBEADM_PATH, assume that "make all..." just built it
   export KUBEADM_PATH="${KUBEADM_PATH:=$(kube::realpath "${KUBE_ROOT}")/_output/local/go/bin/kubeadm}"
+  # the tests should take around ~1 minute, but this ensures there is a higher timeout
+  export KUBE_TIMEOUT=--timeout=240s
   # invoke the tests
```

Comment:
Instead of persist export here, we can just pass this to make the same way we pass WHAT
This should also ideally still have a user override if possible 🤔

Reply:
Hi @neolit123 do you have time to implement @BenTheElder 's feedback? 

I would be happy to help out if help is wanted.

Reply:
@RobertKielty
i'm no longer seeing /cmd/kubeadm/ test failures...are you still seeing them?
i'd prefer to keep the default timeout of 120s if we can, instead of making the change in this PR.

i had a question for @BenTheElder , but looks like i didn't post it.
would `make -C "${KUBE_ROOT}" test WHAT=k8s.io/kubernetes/cmd/kubeadm/test/cmd KUBE_TIMEOUT=--timeout=240s` just work here?